deployment/terraform/assets/dashboard_data.json

```diff
@@ -60,7 +60,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "sum(increase(mattermost_db_store_time_count{method=~\"$top_db_count\"}[1m])) by (method)",
+          "expr": "sum(increase(mattermost_db_store_time_count{method=~\"$top_db_count\"}[$__range])) by (method)",
```

Comment:
If this is a panel instead a table or counter I think is better to use the `$__interval` variable instead of range because interval takes into account the resolution of the graph

Reply:
But they are not equivalent. That graph should show how many API calls we made in the defined time range which value is `$__range`. I am not sure how using `$__interval` there would help getting the correct results.

Reply:
:+1: